--- configure.old	2006-12-25 14:39:26.000000000 +0200
+++ configure	2006-12-25 14:39:35.000000000 +0200
@@ -4609,13 +4609,6 @@
 	GEOS_DIR=`$GEOSCONFIG --prefix`
 
 	GEOS_LDFLAGS=`$GEOSCONFIG --ldflags 2> /dev/null`
-	if test x"$GEOS_LDFLAGS" = "x"; then
-		# older geos-config versions did not
-		# support the --ldflags switch, we'll
-		# default to GEOS_DIR/lib in this case.
-		#
-		GEOS_LDFLAGS="$GEOS_DIR/lib"
-	fi
 
 	GEOS_MAJOR=`$GEOSCONFIG --version | cut -d. -f1`
 	if test "$GEOS_MAJOR" = "@GEOS_VERSION@"; then
